grunt_tasks/plugin.js

```diff
@@ -309,6 +309,12 @@ module.exports = function (grunt) {
             }
 
             var newConfig = webpackHelper(grunt.config.getRaw('webpack.options'), helperConfig);
+            if (_.has(newConfig.module, 'loaders')) {
+                grunt.log.writeln(`  >> "module.loaders" is deprecated, use "module.rules" in ${webpackHelperFile} instead.`.yellow);
+                newConfig.module.rules = newConfig.module.rules || [];
+                newConfig.module.rules = newConfig.module.rules.concat(newConfig.module.loaders);
+                delete newConfig.module.loaders;
+            }
```

Comment:
I tried using the [old webpack helper](https://github.com/girder/girder/blob/master/plugins/dicom_viewer/webpack.helper.js) for the dicom_plugin hit the following error:
```javascript
  >> Loading webpack helper from /home/jbeezley/test/girder/plugins/dicom_viewer/webpack.helper.js
Loading "plugin.js" tasks...ERROR
>> TypeError: Cannot read property 'push' of undefined
```

Could we define `loaders` as an empty array in the config passed to this function?